### PR TITLE
Add datarequest button

### PR DESCRIPTION
**Pull request by andela-onnenanya**
### What does this PR do?

Add `Data Requests` button on the navigation bar

### Which issue does the PR complete:
#26 

### For Authenticated user with `datarequests` plugin `enabled` in config file:
`Data Requests` Button is `visible` on the nav bar

image

### For Unauthenticated user with `datarequests` plugin `enabled` in config file:
`Data Requests` Button is `not visible`

image

### For Authenticated user with `datarequests` plugin `not enabled` in config file:
`Data Requests` Button is `not visible`

image

### For Unauthenticated user with `datarequests` plugin `not enabled` in config file:
`Data Requests` Button is `not visible`

image

### CODE REVIEW

File: ckanext/openafrica/lib/helpers.py [hunk 1]
```diff
@@ -1,5 +1,14 @@
 import datetime
+from ckan.common import config
 
 def current_year():
     year = datetime.datetime.now().year
-    return year    
+    return year
+
+def enable_datarequest():
```
Comment (DavidLemayian) [inline ckanext/openafrica/lib/helpers.py]: I believe this can be better named and easily reusable by having it as `def is_plugin_enabled(plugin_name)`.

Comment (DavidLemayian) [reply ckanext/openafrica/lib/helpers.py]: Also, let's add docstrings about this.

**Review by DavidLemayian — CHANGES_REQUESTED** [review]

File: ckanext/openafrica/lib/helpers.py (hunk 1 above)
Comment (andela-onnenanya) [reply ckanext/openafrica/lib/helpers.py]: Thank you for that. I have fixed it.

**Comment by DavidLemayian** [comment]
Thanks @andela-onnenanya. Could you add a screenshot in the description of what this looks like?

**Comment by andela-onnenanya** [comment]
Ok @DavidLemayian I've added some screenshots. Hope that helps.

**Review by DavidLemayian — APPROVED** [review]

**Pull request merged**